components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-p4 d-bar8 d-bgc-contrast d-fs-200 d-truncate">
+    <template v-if="items.length">
```

Comment:
this component needs to wait for design suggestion. so please wait to look at the design of the component itself.